app/models/reports/bulk_recommend_report.rb

```diff
@@ -103,7 +103,7 @@ def csv_row(trainee)
         trainee_report.course_education_phase,
         trainee_report.course_age_range,
         trainee_report.subjects,
-      ].map { |value| CsvValueSanitiser.new(value).sanitise }
+      ]
```

Comment:
I don't _think_ the sanitiser is the problem in this case, but might be something to review in the future.